### DISCUSSION TITLE
Translations in Leader-Mission mod

### DISCUSSION THREAD

**Discussion by Vitaly220712** [Jiangeshi/Leader-Mission#1] [Ideas]
Please add English translation for Leader-Mission mod.

**Comment by Jiangeshi** [comment]
The English translation of the mod will be carried out gradually. There will be an update today, which will enable all units to have English translations.Thank you for your attention.

**Reply by Tallnut-Enjoyer** [reply]
Are the English translations still happening? I've been playing the mod recently and have been loving it! It would be awesome if the rest of the tech tree, quotes, and descriptions were translated. Thank you!

**Comment by Jiangeshi** [comment]
The mod will complete all English translation in about two weeks
